libcloud/compute/drivers/cloudstack.py

```diff
@@ -1570,14 +1571,24 @@ def create_node(self, **kwargs):
         :type       ex_affinity_groups: ``list`` of
                                         :class:`.CloudStackAffinityGroup`
 
+        :keyword    ex_http_method: HTTP method to use for the request
+                                    (GET or POST). Defaults to "GET"
+        :type       ex_http_method: ``str``
+
         :rtype:     :class:`.CloudStackNode`
         """
 
         server_params = self._create_args_to_params(None, **kwargs)
 
-        data = self._async_request(command='deployVirtualMachine',
-                                   params=server_params,
-                                   method='GET')['virtualmachine']
+        method = kwargs.get('ex_http_method', 'GET')
```

Comment:
Does it make sense to make this user-configurable?

Can we just always use POST, or will this cause issues with some CloudStack installations?